<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sampling</class>
 <widget class="QDialog" name="Sampling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbox_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>UniformSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DownSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RandomSampling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SamplingSurfaceNormal </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NormalSpaceSampling</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
     <item>
      <widget class="QDoubleSpinBox" name="dspin_value">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_askeypoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</normaloff>:/icon/resource/icon/snap-bounding-box-center.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QSpinBox" name="spin_seed"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspin_ratio">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_preview">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/preview.svg</normaloff>:/icon/resource/icon/preview.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/add.svg</normaloff>:/icon/resource/icon/add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/apply1.svg</normaloff>:/icon/resource/icon/apply1.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/reset.svg</normaloff>:/icon/resource/icon/reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/close-copy.svg</normaloff>:/icon/resource/icon/close-copy.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
